Mod0/TechnicalDay1.md

```diff
@@ -0,0 +1,75 @@
+
+## Computer Action
+
+Different computers can perform different actions.  A common action is for a computer to give a user some information.  We call this `output`.  Output is the information that a program or computer gives to a user.
+
+Work through the tutorial linked below to learn about how C# outputs information to users.  Make sure to walk through each `Try it yourself` section.
+* [Outputting Information](https://www.w3schools.com/cs/cs_output.php)
+
+## Computer Observations
+
+Computers make observations by receiving input.  A computer _only_ knows what we input into it; it has no other view of 'the world'.  To learn more about computer observations, work through the tutorials below.  Make sure to walk through each `Try it yourself` Section.
+
+* [Variables](https://www.w3schools.com/cs/cs_variables.php) & [Variables](https://www.w3schools.com/cs/cs_variables_display.php)
+* [Input](https://www.w3schools.com/cs/cs_user_input.php)
+* [Boolean](https://www.w3schools.com/cs/cs_booleans.php)
+* [Comparisons](https://www.w3schools.com/cs/cs_operators_comparison.php)
+
+## Computer Analysis
+
+The final piece of decision making for computers, is an analysis of what to do based on the input it is given.  We need to tell computers exactly what to do with any given piece of information.  Learn more about computer analysis by working through the tutorials below.  Make sure to walk through each `Try it yourself` section.
+
+* [If Statements](https://www.w3schools.com/cs/cs_conditions.php)
+* [Else](https://www.w3schools.com/cs/cs_conditions_else.php)
+* [Else If](https://www.w3schools.com/cs/cs_conditions_elseif.php)
+
+## Putting it all together
+
+In order to create an interesting computer program, we need all three of the steps above to be working together.
+
+Open [this repl](https://replit.com/@MeganMcMahon1/MakingDecisions#main.cs) and complete the following:
+1. Fork the repl so that you have your own copy.
+2. Identify as many of the tools you learned about above as you can.  Make a [comment](https://www.w3schools.com/cs/cs_comments.php) above each line where you see something you recognize.
+3. Run the program by clicking on the green play button!
+
+## Thinking Like a Developer
+
+As a reminder, this session was intended to give you an introduction to some programming concepts - you are not going to master these things yet.
+
+What you have started doing today is practicing what it is like to 'think like a developer'.  Developers must consider _how_ decisions get made and _how_ actions are taken in order to write programs.  Developers break down decisions into small detailed steps, and then write the code to make those steps happen!
+
+## Reflection Questions
+
+In a notebook, reflect on the following:
+1. What is one decision you made today - how did you Observe, Analyze, and Act on that decision?
```

Comment:
Maybe we could do an instructor note to lead this as a chat waterfall to introduce that way of responding? Or maybe there is another better opportunity for that because this is a larger question.